### ISSUE TITLE
Right Panel problem

### ISSUE THREAD

**Issue by Glinthor** [moo-man/WFRP4e-FoundryVTT#2231]
**Describe the bug**
After colapsing, right panel top icons are not visible on the screen. They are beyon it

**To Reproduce**
Colapsing right panel

**Screenshots**
![image](https://github.com/user-attachments/assets/96b626a5-43d3-4780-a7a4-6ad44a3a292f)

**Version Numbers**
PLEASE PROVIDE YOUR FOUNDRY, SYSTEM, and MODULE VERSIONS (where applicable) 

Foundry: 12.331
wfrp4e: 8.0.4.1
wfrp4e-core: 6.0.0
wfrp4e-starter-set:
wfrp4e-rnhd:
wfrp4e-eis:
wfrp4e-ua1:
wfrp4e-dotr:
wfrp4e-archives1:
wfrp4e-middenheim:
wfrp4e-archives1:
wfrp4e-pbtt:
wfrp4e-altdorf:
wfrp4e-ua2:
wfrp4e-owb1:
wfrp4e-horned-rat:
wfrp4e-empire-ruins:

**Comment by moo-man** [comment]
I cannot reproduce this, have you tried disabling modules?

**Comment by Glinthor** [comment]
Yes I traied this. I also tried to uninstal whole Foundry and check only Word with bare system module. Additionly I have exactly the same issue on laptop where i have linux version (latest Ubuntu). All the time I don,t see top icons.

**Comment by Glinthor** [comment]
... and there is one additional thing. On the right panle there is a dice symbol. I assume that it should be clicalbe, but it isin't. It is activetaed only when thir party module for dice is instaled.

**Comment by Forien** [comment]
> ... and there is one additional thing. On the right panle there is a dice symbol. I assume that it should be clicalbe, but it isin't. It is activetaed only when thir party module for dice is instaled.

That button never was clickable in Foundry

> Yes I traied this. I also tried to uninstal whole Foundry and check only Word with bare system module. Additionly I have exactly the same issue on laptop where i have linux version (latest Ubuntu). All the time I don,t see top icons.

Can you check if the same appears when you access your Foundry using Chrome or Edge browsers?

**Comment by Glinthor** [comment]
I'm hosting game as a GM so I'm not using browser. (at lest this is my understending). I have just did some checks and creat two Worlds. One for Call of Cthulhu 7ed. and D&D 5 ed. end evrything is ok for both game systems.

**Comment by moo-man** [comment]
Not applicable in V13
